.drone/pipelines/default/steps/deploy/wait_kubecf.sh

```diff
@@ -27,7 +27,7 @@ while IFS='' read -r line; do instance_groups+=("${line}"); done < <(
     --namespace "${KUBECF_NAMESPACE}" \
     --output jsonpath='{ .data.manifest\.yaml }' \
     | base64 --decode \
-    | yaml2json \
+    | yq \
```

Comment:
Doesn't this need to be
```suggestion
    | yq read --tojson - \
```

Reply:
It does. Thanks for catching this.